src/UserFrontend.php

```diff
@@ -61,13 +61,22 @@ public static function login_head() {
    */
   public static function login_form_defaults(array $args) {
     $args['value_remember'] = TRUE;
+    $args['remember'] = FALSE;
     return $args;
   }
 
   /**
    * @implements login_enqueue_scripts
    */
   public static function login_enqueue_scripts() {
+    echo <<<EOD
+<style>
+#loginform .forgetmenot {
+  display: none;
+}
+</style>
```

Comment:
@sun — nope… login.css is in the theme… in this case SVC but this is a PR for `wordpress-core-standards` plugin so I suppose that change to `login.css` should be applied as commit to the SVC repo, am I wrong? 

However in the SVC case it was already here in commit `df469492`.